cmd/glasskube/cmd/bootstrap_git.go

```diff
@@ -155,10 +205,48 @@ var bootstrapGitCmd = &cobra.Command{
 			}
 		}
 
-		fmt.Fprint(os.Stderr, doneMessage)
+		// try to open argo-cd every 5 seconds until it succeeds in order to check whether it is ready
+		fmt.Fprintf(os.Stderr, "\nglasskube argo-cd application applied successfully!\n\nWaiting for argo-cd to be ready...")
+		argoCheckCtx, cancel := context.WithTimeout(ctx, 5*time.Minute)
+		defer cancel()
+
+		tick := time.NewTicker(5 * time.Second)
+		tickC := tick.C
+		defer tick.Stop()
+	WaitLoop:
+		for {
+			select {
+			case <-argoCheckCtx.Done():
+				fmt.Fprintf(os.Stderr, "\nargo-cd has not become ready in the specified timeout: %v\n", ctx.Err())
+				break WaitLoop
+			case <-tickC:
+				if ready, _ := open.NewOpener().HasReadyPod(argoCheckCtx, argoCdPkg, "", 0); ready {
+					break WaitLoop
+				}
+				// else: generally assume an error means not ready yet
+			}
+		}
+
+		// get initial admin password to print it
+		var adminPwStr string
+		if adminPw, err := clientset.CoreV1().Secrets(argocdNamespace).
+			Get(ctx, "argocd-initial-admin-secret", metav1.GetOptions{}); err != nil {
+			fmt.Fprintf(os.Stderr, "\nCould not obtain initial ArgoCD password: %v\n\n", err)
+		} else {
+			adminPwStr = string(adminPw.Data["password"])
```

Comment:
I think secret data must be deserialized from base64, for example see https://github.com/glasskube/glasskube/blob/71f12d3ba10239d0197aab1a2315f6c970269d69/internal/manifestvalues/resolver.go#L80